vita3k/interface.cpp

```diff
@@ -779,7 +779,7 @@ bool handle_events(EmuEnvState &emuenv, GuiState &gui) {
 
 ExitCode load_app(int32_t &main_module_id, EmuEnvState &emuenv) {
     if (load_app_impl(main_module_id, emuenv) != Success) {
-        std::string message = fmt::format("Failed to load \"{}\"\nSee console output for details.", emuenv.pref_path / "ux0/app" / emuenv.io.app_path / emuenv.self_path);
+        std::string message = fmt::format("Failed to load \"{}\"\nPlease check vita3k.log to see console output for details.\n1. Do you have installed firmware?\n2. Dump your own app(s)/game(s) and install it.\nIf you want to install Vitamin, it is not supported.", emuenv.pref_path / "ux0/app" / emuenv.io.app_path / emuenv.self_path);
```

Comment:
Maybe it would be good to move this string to lang xml file.

Reply:
Yes, you are right, sir.

Reply:
Done.